### PR TITLE
add asdf as an installation method

### PR DESCRIPTION
**Pull request by cometkim**

### CODE REVIEW

File: src/getting_grain.md [hunk 1]
```diff
@@ -50,6 +50,24 @@ curl -LO https://github.com/grain-lang/grain/releases/download/grain-v0.4.6/grai
 
 You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.
 
+## Alternatives
```
Comment (phated) [inline src/getting_grain.md:53]: I'm wondering if we should call this 
```suggestion
## Community
```

And then add a description about community run installation mechanisms and that the Grain team doesn't make sure they are up-to-date.

File: src/getting_grain.md [hunk 2]
```diff
@@ -50,6 +50,24 @@ curl -LO https://github.com/grain-lang/grain/releases/download/grain-v0.4.6/grai
 
 You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.
 
+## Community
+
+### Using ASDF
```
Comment (cometkim) [inline src/getting_grain.md:55]:
```suggestion
### Using asdf
```

File: src/getting_grain.md (hunk 1 above)
Comment (ospencer) [reply src/getting_grain.md]: How about this as a description?

> These installation methods are maintained by the community—they're not endorsed or supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

Comment (phated) [reply src/getting_grain.md]: I think, just by being on the website, they are "endorsed" in a way. Let's just say "they're not supplied by the Grain team, but"

File: src/getting_grain.md [hunk 3]
```diff
@@ -50,6 +50,24 @@ curl -LO https://github.com/grain-lang/grain/releases/download/grain-v0.4.6/grai
 
 You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.
 
+## Community
+
+### Using asdf
```
Comment (phated) [inline src/getting_grain.md:55]:
```suggestion
## Community

These installation methods are maintained by the community—they're not supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

### Using asdf
```

**Review by phated — CHANGES_REQUESTED** [review]
How about adding this note?

File: src/getting_grain.md (hunk 3 above)
Comment (cometkim) [reply src/getting_grain.md:55]: looks great

File: src/getting_grain.md [hunk 4]
```diff
@@ -50,6 +50,26 @@ curl -LO https://github.com/grain-lang/grain/releases/download/grain-v0.4.6/grai
...
+asdf plugin add grain
+
+# List all available version
+asdf list all grain
+
+asdf install grain 0.4.6
+asdf global grain 0.4.6
+```
+
+See the [plugin](https://github.com/cometkim/asdf-grain) for detail.
```
Comment (phated) [inline src/getting_grain.md:71]:
```suggestion
See the [plugin](https://github.com/cometkim/asdf-grain) for more details.
```

Comment (phated) [reply src/getting_grain.md:71]: Just going to commit this and approve

**Pull request merged**